README.md

```diff
@@ -46,48 +46,68 @@ Have a security issue you want to let us know? You can contact us on our website
 
 Our main website is https://polykey.com
 
+## Flakes
+
+Polykey-CLI uses flakes for its build process. By default Nix does not enable this feature.
+To enable it by default, add the following to `~/.config/nix/nix.conf` or `/etc/nix/nix.conf`:
+
+```
+experimental-features = nix-command flakes
+```
+
+Or if you're on NixOS, you can add it to your flakes system configuration:
+
+```
+nix.settings.experimental-features = [ "nix-command" "flakes" ];
+```
+
+Alternatively, to enable it temporarily append the following to any nix related commands:
+
+```
+--extra-experimental-features flakes
+```
+
 ## Installation
 
 Note that JavaScript libraries are not packaged in Nix. Only JavaScript applications are.
 
 Building the package:
 
 ```sh
-npmDepsHash="$(prefetch-npm-deps ./package-lock.json)"
-nix-build -E "(import ./pkgs.nix {}).callPackage ./default.nix { npmDepsHash = \"$npmDepsHash\"; }"
+nix build .#polykey-cli
 ```
 
 ### Nix/NixOS
 
 Building the releases:
 
 ```sh
-nix-build ./release.nix --attr application --argstr npmDepsHash "$(prefetch-npm-deps ./package-lock.json)"
-nix-build ./release.nix --attr docker --argstr npmDepsHash "$(prefetch-npm-deps ./package-lock.json)"
-nix-build ./release.nix --attr package.linux.x64.elf --argstr npmDepsHash "$(prefetch-npm-deps ./package-lock.json)"
-nix-build ./release.nix --attr package.windows.x64.exe --argstr npmDepsHash "$(prefetch-npm-deps ./package-lock.json)"
-nix-build ./release.nix --attr package.macos.x64.macho --argstr npmDepsHash "$(prefetch-npm-deps ./package-lock.json)"
+nix build .#polykey-cli
+nix build .#docker
+
+nix build .#packages.x86_64-linux.polykey-cli
+nix build .#packages.x86_64-windows.polykey-cli
+nix build .#packages.x86_64-darwin.polykey-cli
```

Comment:
Why is there an empty line here? If you're groping them in some fashion then use a comment `# some comment` to explain it in the example. Otherwise I don't see a point to the line.

Reply:
You're right, it should've been made more explicit in the usage. I have added inline comments.